docs/dev_setup.rst

```diff
@@ -48,7 +94,7 @@ Use the fork button to create a libcellml repository under your own account, see
    Fork button for libCellML repository
 
 Clone
-=====
+-----
 
 You now need to clone the libCellML repository to your PC.  You do this by going to your fork (in this example user *andre*'s fork) at::
```

Comment:
You now need to clone the libCellML repository to your computer.  You do this by going to your fork (in this example user *andre*'s fork) at::